editor.planx.uk/src/@planx/components/Calculate/Calculate.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import ErrorOutline from "@mui/icons-material/ErrorOutline";
+import Typography from "@mui/material/Typography";
+import { Meta } from "@storybook/react";
+import React from "react";
+
+import Wrapper from "../fixtures/Wrapper";
+import { WarningContainer } from "../shared/Preview/WarningContainer";
+import Editor from "./Editor";
+import Public from "./Public/index";
+
+export default {
+  title: "PlanX Components/Calculate",
+  component: Public,
+} as Meta;
+
+export const WithEditor = () => {
+  return (
+    <>
+      <Wrapper Editor={Editor} Public={Public} />
+      <WarningContainer>
+        <ErrorOutline />
+        <Typography variant="body2" ml={2}>
+          This component is only available in the Editor when designing
+          services, it does <strong>not</strong> display in the Public form.
+        </Typography>
+      </WarningContainer>
```

Comment:
In editor-only cases, there won't be any user feedback when you click "Update" on the `<Wrapper />` unless there's a validation error in the modal itself. I think manually adding our existing `<WarningContainer />` here is a simple solution to communicate expectations?